src/main/java/org/openRealmOfStars/starMap/Route.java

```diff
@@ -180,10 +113,9 @@ public boolean isFixing() {
    */
   public void makeNextMove() {
     for (int i = 0; i < ftlSpeed; i++) {
-      if (distance > 0) {
-        startX = startX + mx;
-        startY = startY + my;
-        distance--;
+      if (getDistance() > 0) {
+        startX = startX + getMx();
```

Comment:
Okay, I did some mistake somewhere. They are actually same. I'll take one more look for merge request and commit it soon.